### PR TITLE
Late update to correct content for triggers

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR updates the content for mapping triggers for technical accuracy.

### CODE REVIEW

File: downstream/modules/platform/con-gw-authenticator-map-triggers.adoc [hunk 1]
```diff
@@ -10,22 +10,22 @@ Always:: The trigger should always be fired.
...
 
 Attribute:: The map is true or false based on a users attributes coming from the source system. When defining an attribute trigger, the authentication mapping expands to include the following selections:
 +
-* *Operation:* This field includes conditional settings that trigger the handling of the rule based on the specified *Attribute* criteria. In version 2.5.0 this field needs to be set but is ignored. In the future, attribute based maps will allow for the processing of multiple attributes. 
+* *Operation:* This field includes conditional settings that trigger the handling of the rule based on the specified *Attribute* criteria. In version {PlatformVers} this field indicates what will happen if the source system returns a list of  attributes instead of a single value. For example, if the source system returns multiple emails for a user and *Operation* was set to *and*, all of the given emails must match the Comparison for the trigger to be _True_. If *Operation* was set to *or*, any of the returned emails will set the trigger to _True_ if they match the *Comparison* in the trigger.
```
Comment (sayjadha) [inline downstream/modules/platform/con-gw-authenticator-map-triggers.adoc:24]: Why is 'Comparison' in title case? Is it a UI field? If it is, then it should also be in bold formatting.

**Review by sayjadha — COMMENTED** [review]
One small comment, rest LGTM.

File: downstream/modules/platform/con-gw-authenticator-map-triggers.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-gw-authenticator-map-triggers.adoc:24]: Good catch! Fixed.

Comment (sayjadha) [reply downstream/modules/platform/con-gw-authenticator-map-triggers.adoc:24]: @dcdacosta : I don't see it updated in this instance:
given emails must match the Comparison for the trigger to be _True_.

File: downstream/modules/platform/con-gw-authenticator-map-triggers.adoc [hunk 2]
```diff
@@ -10,26 +10,26 @@ Always:: The trigger should always be fired.
...
+If you would like to experiment with multiple attribute maps you can do that through the API but the UI form will remove multi-attribute maps if the authenticator is saved through the UI. When adding multiple attributes to a map, the *Operation* will also apply to the attributes.
 ====
 +
 * *Attribute:* The name of the attribute coming from the source system this trigger will be evaluated against. For example, if you wanted the trigger to fire based on the user's last name and the last name field in the source system was called `users_last_name` you would enter the value ‘users_last_name’ in this field.
-* *Comparison:* Tells the trigger how to evaluate the value of the users. *Attribute* in the source system compared to the *Value* specified on the trigger. Available options are: *contains*, *matches*, *ends with*, *in*, or *equals*. Below is a breakdown of each comparison type:
+* *Comparison:* Tells the trigger how to evaluate the value of the users. *Attribute* in the source system compared to the *Value* specified on the trigger. Available options are: *contains*, *matches*, *ends with*, *in*, or *equals*. Below is a breakdown of each *comparison* type:
```
Comment (sayjadha) [inline downstream/modules/platform/con-gw-authenticator-map-triggers.adoc:32]: **comparison** -> **Comparison**

**Review by sayjadha — COMMENTED** [review]
A few comments please

File: downstream/modules/platform/con-gw-authenticator-map-triggers.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/con-gw-authenticator-map-triggers.adoc]: Fixed.

**Review by sayjadha — APPROVED** [review]
LGTM.

**Pull request merged**